### DISCUSSION TITLE
[Feature] Avoid showing two identical images when splitview enabled

### DISCUSSION THREAD

**Discussion by dimatx** [damongolding/immich-kiosk#130] [Feature request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Building on top of https://github.com/damongolding/immich-kiosk/discussions/17, hoping you could consider adding logic to avoid showing the exact same image side by side. I understand you're not looking to add additional complexity such as https://github.com/damongolding/immich-kiosk/discussions/111 at this point, but perhaps avoiding two identical images side by side is more straightforward to achieve.

**Comment by damongolding** [comment]
Are you seeing this from a specific data source? I.e. album, person or random? And is said that data source small?

**Reply by dimatx** [reply]
From an album source. 225 photos.

But as I was checking that, I am noticed that there are some duplicates in the album due to migrating multiple users from Google Photos, and then creating a shared album... Let me tackle that first, and comment back if still an issue I'm noticing.

**Reply by Bhuemann** [reply]
Ive noticed this happening too. Its pretty easy to reproduce with single albums with few assets

**Reply by damongolding** [reply]
It's kind of expected with a small image source. I can add some protection against the same image display though.

**Reply by dimatx** [reply]
Thanks @damongolding. Yeah, my issue was primarily driven by actually having duplicates, but agree it's good to have protection against the same image showing up twice assuming there is an alternative candidate. Thanks!

**Comment by damongolding** [comment]
Patched in 0.11.3
